docs/services/navigation/_index.md

```diff
@@ -115,6 +119,54 @@ The following attributes are available for `Navigation` services:
+
+It stores the required contextual information for Viam's services like [Motion](/services/motion/) and [Vision](/services/vision/) to use the position and orientation readings returned by components like [movement sensors](/components/movement-sensor/).
+
+{{% /alert %}}
+
+For the most accurate autonomous GPS navigation with the Navigation Service, configure and calibrate the Frame System Service for the components of your robot.
+To start, add the Frame System Service to your rover [base](/components/base/) and [movement sensor](/components/movement-sensor/).
+
+- Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+  Scroll to the card with the name of your base:
+
+    {{< imgproc src="/services/navigation/click-add-frame-ui.png" alt="The button to add a frame selected with the cursor on the Viam App config builder." resize="500x" >}}
+
+- Click **Add Frame**.
+
+  - Keep the `parent` frame as `world`.
+  Select the **Geometry** drop-down menu.
+  - Configure a **Geometry** for the base that reflects its physical dimensions.
+  Reference [these instructions](/services/frame-system/#bounding-geometries) to configure your geometry and measure the physical dimensions of your base.
+
+    ![The frame card for the base in the Viam App config builder.](/services/navigation/full-frame-base-ui.png)
+
+- Scroll to the card with the name of your movement sensor.
+  Click **Add Frame** and select the **Parent** box.
+  - Type in the `name` of your base to specify this component as the `parent` of the sensor in the reference frame coordinate system, and click **Save Config** to save your configuration.
+  See [how to configure nested reference frames](/services/frame-system/nested-frame-config/) for an explanation of this configuration process.
+  ![The frame card for the base in the Viam App config builder.](/services/navigation/full-frame-movement-sensor-ui.png)
+
+  - Give the movement sensor a **Translation** that reflects where it is mounted on your base, measuring the coordinates with respect to the origin of the base.
+
+    In other words, designate the origin of the base as `(0,0,0)`, and measure the distance the sensor is *translated*, or, *displaced*, from that point, to obtain the coordinates of the **Translation**.
+
+    See [the Frame System Service](/services/frame-system/#configuration) for more information, and [Viam's Internals](/internals/orientation-vector/) for a detailed guide on conducting this measurement.
+
+Then, to calibrate your Frame System for the most accurate autonomous GPS navigation with the Navigation Service:
+
+- After configuring your robot, navigate to the **Control** page and select the card matching the name of your Navigation Service.
+- Monitor the readings displayed on the card, and verify that the compass or orientation readings from the movement sensor report `0` when the base is facing north.
```

Comment:
I don't believe the compass or orientation readings will show up on the Navigation Service card. Once https://viam.atlassian.net/browse/RSDK-4140 is implemented, you'll be able to see the compass heading in the Navigation Service card, but this is not available yet. For now, the sensor readings will show up in the movement sensor card.

Reply:
Ok, thank you! Updating to reflect